File: christmas/december task 2/function tasks/IsPrime.js
const prompt = require('prompt-sync')();

function isPrime(number){


  number = prompt("Enter any Integer");
     

    factor = 0;
    result = 0;

    for(count = 2; count <= number; count ++) {
        if(number % count == 0) {
          result = number / count;
            factor += count;
            
            
            if(result  == 1) {
               return true;                
            }

             return false;
             }

            
            }

           

                
               
    }


number = 0;

console.log(isPrime(number));

File: christmas/december task 2/function tasks/IsPrime.java
import java.util.Scanner;
    public class IsPrime {
        public static void main(String...Odili) {
           Scanner input = new Scanner(System.in);

           int number = 0;
           System.out.print(isPrime(number));

        }
            

            public static boolean isPrime(int number){

                Scanner input = new Scanner(System.in);

                System.out.println("Enter any Integer");
                  number = input.nextInt();

                int factor = 0;
                int result = 0;

                for(int count = 2; count <= number; count ++) {
                    if (number % count == 0) {
                        result = number / count;
                        factor += count;

                    }
                    if (result == 1) {
                        return true;
                    }
                }

                   return false;


                     }



            }

